fhirAssembly/src/main/resources/assembly/com.vantiq.fhir.fhirConnection/procedures/com/vantiq/fhir/fhirService_fetchSMARTConfiguration.vail

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
FHIR servers define some well-known places to pick up configuration information.  Provide built-in means to fetch these.